patch to stop disabling the IoT interface, thanks to Mel:
https://forum.openwrt.org/t/xiaomi-ax3600-int-wifi/74276/523

-----
--- a/lib/wifi/qcawificfg80211.sh	2021-07-08 06:48:54.000000000 +0000
+++ b/lib/wifi/qcawificfg80211.sh	2022-07-10 17:08:27.153316389 +0000
@@ -3039,10 +3039,6 @@
 		disable_qrfs
 	fi
 
-	if [ $ifname = "wl2" ]; then
-		ifconfig $ifname down
-	fi
-	
 	#need to check router bind or not
 	if [ $ifname == "wl13" ] && [ $bindstatus == 0 -o $userswitch == 0 ];then
 		hostapd_cli -i wl13 -p /var/run/hostapd-wifi1 disable
